build.sh

```diff
@@ -11,19 +11,29 @@
 BASE_OS_IMAGE="${BASE_OS_IMAGE:-adoptopenjdk/openjdk11-openj9:ubi}"
 
 #
-# Version of Open Liberty runtime to use within both inner and outer loops
+# Version of Liberty runtime to use within both inner and outer loops
 #
-OL_RUNTIME_VERSION="${OL_RUNTIME_VERSION:-21.0.0.3}"
+LIBERTY_RUNTIME_VERSION="${LIBERTY_RUNTIME_VERSION:-21.0.0.3}"
 
 #
-# The Open Liberty base image used in the final stage of the outer loop Dockerfile used to build your application image from
+# Archive Id of the Liberty runtime archive
 #
-OL_UBI_IMAGE="${OL_UBI_IMAGE:-openliberty/open-liberty:21.0.0.3-full-java11-openj9-ubi}"
+LIBERTY_RUNTIME_ARTIFACTID="${LIBERTY_RUNTIME_ARTIFACTID:-openliberty-runtime}"
+
+#
+# Group Id of the Liberty runtime archive
+#
+LIBERTY_RUNTIME_GROUPID="${LIBERTY_RUNTIME_GROUPID:-io.openliberty}"
+
+#
+# The Liberty base image used in the final stage of the outer loop Dockerfile used to build your application image from
+#
+LIBERTY_IMAGE="${LIBERTY_IMAGE:-openliberty/open-liberty:21.0.0.3-full-java11-openj9-ubi}"
 
 #
 # The name and tag of the "stack image you will build.  This will used to create your inner loop development containers, and also as the base image for the first stage of your outer loop image build.
 #
-STACK_IMAGE="${STACK_IMAGE:-openliberty/application-stack:0.5}"
+STACK_IMAGE="${STACK_IMAGE:-openliberty/application-stack:0.6}"
```

Comment:
The dropping the build command is is a substantial change.   I'm not 100% sure though that the old devfile wouldn't work with the new stack image or vice versa.    But maybe it's better to call it 0.6  anyway.

Reply:
The older devfile will try to call devbuild-cmd.sh which wont be there in the new stack-image. So this will be a breaking change.